sky/templates/gcp-ray.yml.j2

```diff
@@ -59,8 +59,13 @@ available_node_types:
       guestAccelerators:
         - acceleratorType: projects/{{gcp_project_id}}/zones/{{zones}}/acceleratorTypes/{{gpu}}
           acceleratorCount: {{gpu_count}}
+  {%- endif %}
       metadata:
         items:
+          - key: ssh-keys
+            value: |
+              skypilot:ssh_user:skypilot:ssh_public_key_content
```

Comment:
After replacement does this become `me:<my pub key content>`? Is this the expected syntax? Worth a comment.